src/commands/push.js

```diff
@@ -20,5 +24,4 @@ function action(options) {
   runWithTruffle(async () => await push({ network, deployStdlib, reupload, txParams }), network, ! skipCompile)
 }
 
-module.exports = registerPush
-module.exports.action = action
+module.exports = { signature, description, setup: registerPush, action}
```

Comment:
Nit: I'd prefer being consistent with how we export modules, I vote to do this way. I think it's clearer to define variables & functions above and export it at the end of the file.